cli/src/commands/start-prover/index.ts

```diff
@@ -9,25 +9,24 @@ class StartProver extends Command {
   }
 
   static flags = {
-    "skip-prove-compressed-accounts": Flags.boolean({
-      description: "Skip proving of compressed accounts.",
-      default: false,
-      char: "c",
-    }),
-    "skip-prove-new-addresses": Flags.boolean({
-      description: "Skip proving of new addresses.",
-      default: false,
-      char: "n",
-    }),
     "prover-port": Flags.integer({
       description: "Enable Light Prover server on this port.",
       required: false,
       default: 3001,
     }),
     "run-mode": Flags.string({
-      description: "Specify the running mode (test or full)",
-      options: ["test", "full"],
-      default: "full",
+      description:
+        "Specify the running mode (forester, forester-test, rpc, full, or full-test).",
+      options: [
+        "inclusion",
+        "non-inclusion",
+        "rpc",
+        "forester",
+        "forester-test",
```

Comment:
can we change test mode to use height 26 as well?
(I just changed that on my batched update branch because testing with height 10 doesn't make sense for those rust tests.)